welcomescreen.py

```diff
@@ -226,7 +226,9 @@ def get_sonic_pos():
                 self.showing_help = False
 
             # Help Button Action
-            if help_btn.collidepoint(mos_x, mos_y) and (pygame.mouse.get_pressed())[0] == 1 and press == 0 and not self.showing_help:
+            if help_btn.collidepoint(mos_x, mos_y) and \
+               (pygame.mouse.get_pressed())[0] == 1 and \
+               press == 0 and not self.showing_help:
```

Comment:
The indentation here is wrong, it should be in a way that the block below the if statement shouldn't be difficult to find.

Reply:
```
            if help_btn.collidepoint(mos_x, mos_y) and \
                                    (pygame.mouse.get_pressed())[0] == 1 and \
                                    press == 0 and not self.showing_help:
                self.showing_help = True
```

Is this a correct way?

Reply:
The indentation should raise flake8 errors so no, something like this;

```python3
if help_btn.collidepoint(mos_x, mos_y) and \
        (pygame.mouse.get_pressed())[0] == 1 and \
        press == 0 and not self.showing_help:
    self.showing_help = True
```

Reply:
Done